client_vehiclespecification.go

```diff
@@ -0,0 +1,111 @@
+package mbz
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+
+	mbzv1 "github.com/way-platform/mbz-go/proto/gen/go/wayplatform/connect/mbz/v1"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+// GetVehicleSpecificationRequest is the request for [Client.GetVehicleSpecification].
+type GetVehicleSpecificationRequest struct {
+	// VehicleID is the FIN or VIN of the vehicle (17 characters).
+	VehicleID string
+
+	// Locale is the market locale for country and language (e.g., "en_US", "de_DE").
+	// Required parameter.
+	Locale string
+
+	// OptionsNullDescription when set, controls whether to omit option codes with null descriptions.
+	// Defaults to false.
+	OptionsNullDescription *bool
```

Comment:
avoid pointers in the Go struct requests - these are meant to be as ergonomic as possible